Velaptor/PublicExtensionMethods.cs

```diff
@@ -261,5 +288,280 @@ public static GlyphMetrics ApplySize(this GlyphMetrics value, float size)
 
             return value;
         }
+
+        /// <summary>
+        /// Returns a value indicating if the character is a letter.
+        /// </summary>
+        /// <param name="character">The character to check.</param>
+        /// <returns>True if the character is an upper or lower case letter.</returns>
+        public static bool IsLetter(this char character) => Letters.Contains(character);
+
+        /// <summary>
+        /// Returns a value indicating if the character is not a letter.
+        /// </summary>
+        /// <param name="character">The character to check.</param>
+        /// <returns>True if the character is not an upper or lower case letter.</returns>
+        public static bool IsNotLetter(this char character) => !Letters.Contains(character);
+
+        /// <summary>
+        /// Returns a value indicating if the given <see langword="string"/> <paramref name="path"/>
+        /// is a fully qualified path.
+        /// </summary>
+        /// <param name="path">The path to check.</param>
+        /// <returns><c>true</c> if valid.</returns>
+        /// <example>
+        /// <list type="number">
+        ///     <item>Value of <c>null</c> will return <c>false</c>.</item>
+        ///     <item><c>null</c> or <c>empty</c> will returns <c>false</c>.</item>
+        ///     <item>
+        ///         Start with a drive letter
+        ///         <para>Value of <c>C:/</c> will return <c>true</c>.</para>
+        ///     </item>
+        ///     <item>
+        ///         Contain at least 1 directory
```

Comment:
Grammar
```suggestion
        ///         Contains at least 1 directory.
```
This is also very indented.